libr/magic/file.h

```diff
@@ -54,33 +54,32 @@ typedef unsigned int ssize_t;
 typedef unsigned long unichar;
 
 struct stat;
-const char *file_fmttime(unsigned int, int, char *);
-int file_buffer(struct r_magic_set *, int, const char *, const void *, size_t);
-int file_fsmagic(struct r_magic_set *, const char *, struct stat *);
-int file_pipe2file(struct r_magic_set *, int, const void *, size_t);
-int file_printf(struct r_magic_set *, const char *, ...);
-int file_reset(struct r_magic_set *);
-int file_tryelf(struct r_magic_set *, int, const unsigned char *, size_t);
-int file_zmagic(struct r_magic_set *, int, const char *, const ut8*, size_t);
-int file_ascmagic(struct r_magic_set *, const unsigned char *, size_t);
-int file_is_tar(struct r_magic_set *, const unsigned char *, size_t);
-int file_softmagic(struct r_magic_set *, const unsigned char *, size_t, int);
-struct mlist *file_apprentice(struct r_magic_set *, const char *, size_t, int);
-ut64 file_signextend(RMagic *, struct r_magic *, ut64);
-void file_delmagic(struct r_magic *, int type, size_t entries);
-void file_badread(struct r_magic_set *);
-void file_badseek(struct r_magic_set *);
-void file_oomem(struct r_magic_set *, size_t);
-void file_error(struct r_magic_set *, int, const char *, ...);
-void file_magerror(struct r_magic_set *, const char *, ...);
-void file_magwarn(struct r_magic_set *, const char *, ...);
-void file_mdump(struct r_magic_set *, struct r_magic *);
-void file_showstr(FILE *, const char *, size_t);
-size_t file_mbswidth(const char *);
-const char *file_getbuffer(struct r_magic_set *);
-ssize_t sread(int, void *, size_t, int);
-int file_check_mem(struct r_magic_set *, unsigned int);
-int file_looks_utf8(const unsigned char *, size_t, unichar *, size_t *);
+R_IPI const char *r_magic_file_fmttime(unsigned int, int, char *);
+R_IPI int r_magic_file_buffer(struct r_magic_set *, int, const char *, const void *, size_t);
```

Comment:
use RMagic * instead of `struct r_magic_set`.. will be good to have some consistency across all those functions too